akka-aws/src/main/scala/com/sclasen/akka/aws/AkkaAWSClient.scala

```diff
@@ -95,7 +92,7 @@ abstract class AkkaAWSClient(props: AkkaAWSClientProps) {
     }).mkString("&")
 
   def formData[T](awsReq: Request[T]) =
-    HttpEntity(MediaTypes.`application/x-www-form-urlencoded`, encodeQuery(awsReq))
+    HttpEntity(ContentType(MediaTypes.`application/x-www-form-urlencoded`, HttpCharsets.`ISO-8859-1`), encodeQuery(awsReq))
```

Comment:
here I am using `HttpCharsets.`ISO-8859-1` because is the default defined by the HTTP standard.